### PR TITLE
Commit i-Pay Secure Payments logo

### PR DESCRIPTION
**Pull request by FrancoisVE**
** Checklist **
- [x] All icons include `width="38" height="24"` in their attributes
- [x] All icons have solid white backgrounds
- [x] All icons have a a `viewBox` attribute
- [x] All fonts have been converted to paths
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I am confident that all icons are clear and easy to read/understand

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/i_pay_secure_payments.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg viewBox="211.13499450683594 193 38 24" width="525" height="169.00006103515625" xmlns="http://www.w3.org/2000/svg">
```
Comment (sunblaze) [inline app/assets/images/payment_icons/i_pay_secure_payments.svg]: Isn't the requested width="38" height="24"

Comment (FrancoisVE) [reply app/assets/images/payment_icons/i_pay_secure_payments.svg]: Noted thank you. Will advise designers to amend the issue.

**Comment by FrancoisVE** [comment]
Hi James,

Noted and thank you for the feedback.

We will amend the issues ad re-submit.

Kind Regards
Francois van Eeden
Mobile: +27 060 503 7037
Email: vaneedenfrancois@outlook.com<http://outlook.com>

________________________________

**Comment by NathanPJF** [comment]
Going to close this PR for inactivity. 

Feel free to open a new pull request if you still want to add this icon.  Please note we've made changes to the contributing guidelines. https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md

**Pull request closed without merging**